src/lib/core/components/visualizations/implementations/MosaicVisualization.tsx

```diff
@@ -616,22 +616,20 @@ function MosaicViz(props: Props<Options>) {
   const classes = useInputStyles();
 
   /**
-   * Disabled because reference value selection options are based on the variable's vocabulary
-   * */
-  const areQuadrantSelectionsDisabled =
-    !xAxisVariable?.vocabulary || !yAxisVariable?.vocabulary;
-
-  /**
-   * TEMPORARY: would be better to upgrade CoreUI's SingleSelect (and other selectors) to enable disabling
-   * By using pointerEvents: 'none', we lose the ability to convey messages via tooltips and cursors
-   * */
-  const twoByTwoQuadrantStyle: React.CSSProperties | undefined = !isTwoByTwo
-    ? undefined
-    : {
-        ...twoBytwoInputStyle,
-        pointerEvents: areQuadrantSelectionsDisabled ? 'none' : undefined,
-        opacity: areQuadrantSelectionsDisabled ? 0.5 : 1,
-      };
+   * TEMPORARY:
+   * CoreUI's selection components need to be updated to enable disabling them. We lose the ability to convey messages via tooltips
+   * and cursors when we use pointerEvents: 'none'
```

Comment:
do we have an issue for this already in coreui?

Reply:
Yes! https://github.com/VEuPathDB/CoreUI/issues/142